deploy/kubernetes/base/controller.yaml

```diff
@@ -39,6 +39,8 @@ spec:
             - --logtostderr
             - --v=2
           env:
+            - name: CSI_DRIVER_NAME
```

Comment:
Why we want to expose the name as env variable? Feel like it would be more accurate if the useragent is fixed..

Reply:
yeah I agree, better to just keep it fixed/hardcoded

Reply:
my thinking was that if the driver's name changed for whatever reason, we'd only have to change it in one place. i don't have a strong preference here, so i'll just hardcode it.